cli/cmd/seed/generate_test.go

```diff
@@ -24,47 +24,18 @@ func TestSeedGenerate(t *testing.T) {
 		require.NotNil(t, actualOutput)
 	})
 
-	t.Run("Successfully Generate Mnemonic", func(t *testing.T) {
+	t.Run("Successfully Generate Seed from Mnemonic", func(t *testing.T) {
 		var output bytes.Buffer
 		cmd.ResultPrinter = printer.New(&output)
 		cmd.RootCmd.SetArgs([]string{
 			"seed",
 			"generate",
-			"--mnemonic",
+			"--mnemonic=bone dawn produce network shock transfer magic moment dignity grunt must doll combine olive expose artwork wool wrestle pitch range leg install flip coffee",
```

Comment:
The value should be quoted

Reply:
quoted value received as double quoted value in test.
In real cli command the value should be quoted